lib/core/src/display/world.rs

```diff
@@ -267,6 +266,14 @@ impl World {
         self.rc.borrow_mut().run();
     }
 
+    pub fn event_loop(&self) -> EventLoop {
+        self.rc.borrow().event_loop.clone()
+    }
+
+    pub fn scene<F:FnMut(&Scene)>(&self, mut f:F) {
+        f(&self.rc.borrow().scene)
+    }
```

Comment:
This is very unsafe function, as it is easy to mutable borrow mutliple times. `Scene` is a shared reference object, create instead function `scene(&self) -> Scene`.